util/chunk/row_container.go

```diff
@@ -71,6 +68,11 @@ func (c *RowContainer) SpillToDisk() {
 	if c.alreadySpilled() {
 		return
 	}
+	if c.actionSpill != nil {
+		atomic.StoreUint32(&c.actionSpill.status, 1)
```

Comment:
I think `status` should be locked in the same Locker of `sync.Cond`.
After that `status` can be changed without `atomic`.